src/rules/function-calc-no-unspaced-operator/__tests__/index.js

```diff
@@ -65,6 +65,18 @@ testRule(rule, {
   }, {
     code: "a { padding: rem-calc(10px+ 10px) rem-calc(10px+ 10px); }",
     description: "ignore function have calc in name",
+  }, {
```

Comment:
I see you've account for it in the rule code, but is it also worth adding a `\r\n` tests as well?